README.md

```diff
@@ -17,8 +17,10 @@ The training process consists of the following steps:
 5. Save the model.
 
 ## The end goal
```

Comment:
I think this is tutorial-specific info. maybe move it into the readme of each tutorial. Then the main readme can be just an overview of the tutorials. 
Optionally: move the compute instance creation into a separate tutorial